<ui version="4.0" >
 <class>MailLeecher</class>
 <widget class="QMainWindow" name="MailLeecher" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="resources.qrc" >:/resources/images/mailleecher.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>POP3 server address:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="ServerAddress_" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Mail server port:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QSpinBox" name="ServerPort_" >
            <property name="minimum" >
             <number>1</number>
            </property>
            <property name="maximum" >
             <number>65535</number>
            </property>
            <property name="value" >
             <number>110</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Output directory:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLineEdit" name="OutputDirectory_" >
              <property name="minimumSize" >
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="OutputBrowse_" >
              <property name="text" >
               <string>Browse...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QGridLayout" >
            <item row="0" column="0" >
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>Login:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QLineEdit" name="Login_" >
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>Password:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QLineEdit" name="Password_" >
              <property name="text" >
               <string/>
              </property>
              <property name="echoMode" >
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QPushButton" name="Leech_" >
              <property name="text" >
               <string>Leech!</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Cancel_" >
              <property name="text" >
               <string>Cancel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Messages:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QProgressBar" name="MessagesBar_" >
        <property name="maximum" >
         <number>0</number>
        </property>
        <property name="value" >
         <number>0</number>
        </property>
        <property name="format" >
         <string>%p% (%v from %m)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Data:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QProgressBar" name="DataBar_" >
        <property name="maximum" >
         <number>0</number>
        </property>
        <property name="value" >
         <number>0</number>
        </property>
        <property name="format" >
         <string>%p% (%v bytes from %m)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources>
  <include location="resources.qrc" />
 </resources>
 <connections/>
</ui>
